drivers/mpl3115a2/include/mpl3115a2_reg.h

```diff
@@ -18,8 +18,8 @@
  *
  */
 
-#ifndef __MPL3115A2_REG_H__
-#define __MPL3115A2_REG_H__
```

Comment:
This is not correct. At no point in an identifier two consecutive underscores are allowed.http://stackoverflow.com/a/228797/416224